crates/providers/blobstore-fs/Cargo.toml

```diff
@@ -15,7 +18,9 @@ status = "actively-developed"
 async-trait = { workspace = true }
 path-clean = { workspace = true }
 serde = { workspace = true }
+serde_bytes = { workspace = true }
```

Comment:
where is this used?

Reply:
it's a requirement that becomes used [during bindgen](https://github.com/wasmCloud/wasmCloud/pull/772/files#diff-c35e6db0016f48f0c6fe14869e51b1fbffd9c6d8db1be7933222b5d519e6045aR1608), because of [how the host currently expects binary data to be provided](https://github.com/wasmCloud/wasmCloud/blob/main/crates/compat/src/blobstore.rs#L15), which is [the more efficient way to pass bytes](https://docs.rs/serde_bytes/latest/serde_bytes/)

Reply:
I see. Does not have to happen in this PR, but let's at least file an issue to get rid of the external dependency here and instead re-export and use `serde_bytes` from the bindgen crate itself here

Reply:
That would make this macro unhygenic -- generally we want our downstreams to choose the versions of the libs that we require. It's not like it's impossible, but if we risk incompatibility with a different dep (like `serde`) here if we do that. 

@thomastaylor312 should be able to cross-check this

Reply:
From the guide you linked to, @vados-cosmonic:

> Procedural macros are _unhygienic_

Which seems to assert that "lack of hygiene" is an inherent property of all procedural macros.

Regardless, I don't really see how this is relevant here - I do not think that we should require users of this macro to add custom opaque dependencies - relying on re-exported dependencies is a pretty common practice in the ecosystem, which eliminates the guesswork and ensures compatibility.

Reply:
Filed an issue